.gitignore

```diff
@@ -107,6 +107,9 @@ bubbles.pkl
 missions.pkl
 traffic_histories.pkl
 **/social_agents/*
+*.pkl
```

Comment:
I think I agree with this filter but if we add this then I think we can remove the more specific `.pkl` filters (e.g. `missions.pkl`.)